SmartDeviceLink/SDLRdsData.h

```diff
@@ -0,0 +1,86 @@
+//
+//  SDLRdsData.h
+//
+
+#import "SDLRPCMessage.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface SDLRdsData : SDLRPCStruct
+
+- (instancetype)initWithProgramService:(nullable NSString *)PS;
+
+- (instancetype)initWithRadioText:(nullable NSString *)RT;
+
+- (instancetype)initWithClockText:(nullable NSString *)CT;
+
+- (instancetype)initWithProgramIdentification:(nullable NSString *)PI;
+
+- (instancetype)initWithProgramType:(nullable NSNumber<SDLInt> *)PTY;
+
+- (instancetype)initWithTrafficProgramIdentification:(nullable NSNumber<SDLBool> *)TP;
+
+- (instancetype)initWithTrafficAnnouncementIdentification:(nullable NSNumber<SDLBool> *)TA;
+
+- (instancetype)initWithRegion:(nullable NSString *)REG;
+
+/**
+ * @abstract Program Service Name
+ *
+ * optional, 0-8 length
+ */
+@property (nullable, strong, nonatomic) NSString *PS;
+
+/**
+ * @abstract Radio Text
+ *
+ * optional, 0-64 length
+ */
+@property (nullable, strong, nonatomic) NSString *RT;
+
+/**
+ * @abstract The clock text in UTC format as YYYY-MM-DDThh:mm:ss.sTZD
+ *
+ * optional, 24 length
```

Comment:
Clean up the "length"s, sometimes it says 0-something, sometimes it says it related to an integer. Doesn't always make sense.